frontend/src/Components/cards/HelpfulLinkCard.tsx

```diff
@@ -69,6 +92,19 @@ export default function HelpfulLinkCard({
                         />
                     </div>
                 )}
+            {!AdminRoles.includes(role ?? UserRole.Student) && (
+                <div
+                    className="right-1 top-1 z-100 favorite-toggle"
+                    onClick={(e) => void toggleLinkFavorite(e)}
+                >
+                    <ULIComponent
+                        tooltipClassName="absolute right-2 top-2 z-100"
+                        iconClassName={`w-6 h-6 cursor-pointer ${link.is_favorited ? 'text-primary-yellow' : ''}`}
+                        icon={link.is_favorited ? StarIcon : StarIconOutline}
+                    />
+                </div>
+            )}
+
```

Comment:
make sure you dont render the star on the admin view of the helpful links
<img width="326" alt="Screenshot 2024-12-19 at 10 08 40 AM" src="https://github.com/user-attachments/assets/e4283de6-3583-418a-acb5-3f2f146fb236" />